dutch/over-het-hek/sleutel/nacht/deze-nacht.md

```diff
@@ -1 +1,9 @@
 Plotseling komen vage beelden naar boven ...
+
+Je beseft dat je droomt, alsof je in een slecht verhaal zit, en ziet dat alles verkeerd gespeld is. 
+Een boom heet plotseling booom en benzinetankstation heet bedritanktontiats. 
+
+Maar zoals in alle dromen die van zichzelf bewust zijn 
+heb je de keuze om [wakker te worden](../dutch/welkom.md).
+
+Of toch maar lekker [verder droooomen](../dutch/droom/droom.md).
```

Comment:
I believe here the link `../../../droom/droom.md` would be correct (and a similar change for the other link).